src/CaptureClient/include/CaptureClient/AppInterface.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_CLIENT_APP_INTERFACE_H_
+#define CAPTURE_CLIENT_APP_INTERFACE_H_
+
+#include "CaptureClient.h"
+
+namespace orbit_capture_client {
+
+class AppInterface {
+ public:
+  virtual ~AppInterface() = default;
+
+  [[nodiscard]] virtual CaptureClient::State GetCaptureState() const = 0;
+  [[nodiscard]] virtual bool IsCapturing() const = 0;
+
+  virtual void StartCapture() = 0;
+  virtual void StopCapture() = 0;
+  virtual void AbortCapture() = 0;
```

Comment:
Are these methods needed in the `CaptureWindow` unit tests? But it seems that they are invoked from `CaptureWindow`.

Reply:
I thought they were, but now that you mention it - only `ToggleCapture` is used. But as I commented above, I wanted to create an interface with a somewhat narrow, but complete scope ("Control the capture") and not only limit it to how `CaptureWindow` is interacting with OrbitApp. Does that make sense to you?

Reply:
SGTM! Thanks for the explanation!